config/dependencies/gateway-api/gateway/gateway.yaml

```diff
@@ -8,12 +8,9 @@ metadata:
 spec:
   gatewayClassName: istio
   listeners:
-  - name: default
```

Comment:
I don't mind the name change, but any specific reason to change it?

Reply:
This was just to match the naming when retrieving the port from the gateway as used in the [istio docs](https://istio.io/latest/docs/examples/bookinfo/#determine-the-ingress-ip-and-port) - no real reason though we can leave it as default and I can update the exports in each of the guides